src/include/duckdb/execution/merge_sort_tree.hpp

```diff
@@ -172,6 +201,52 @@ struct MergeSortTree {
 		}
 		return level;
 	}
+
+	void Print() const {
```

Comment:
Is this for debugging? Should we wrap it in `#ifdef DEBUG` to make this more apparent?

Reply:
We have other `Print` methods that are not for debugging only. And when the operations are slow it is nice to be able to print out complex data structures in release.